### PR TITLE
initial stab at age verification documentation

### PR DESCRIPTION
**Pull request by mickhansen**

### CODE REVIEW

**Comment by mickhansen** [comment]
@nmoskaleva at your convenience please try out this guide, and then help make the guide better :)

File: src/pages/verify/guides/age-verification.mdx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+## Integration
+
+### Parameters
+
+Integrating age verification is done via our [standard OpenID Connect offering](/verify/getting-started/oidc-intro/)
+with the follow specifics:
+
+- `acr_values=urn:age-verification`
+- `login_hint=country:{country}`, `{country}` can be one of DK|SE|NO|FI
+- `scope=openid is_over_18`, values such as `is_over_16` and `is_over_18` are also supported, see extension documentation for full list.
```
Comment (jlndk) [inline src/pages/verify/guides/age-verification.mdx:52]:
```suggestion
- `scope=openid is_over_18`, values such as `is_over_16` and `is_over_21` are also supported, see extension documentation for full list.
```

File: src/utils/auth-methods.ts [hunk 2]
```diff
@@ -115,5 +115,21 @@ export const PROVIDERS : Provider[] = [
...
     ],
     page: '/verify/e-ids/belgian-eid'
+  },
+  {
+    title: 'Age verification',
+    authMethods: [
+      {
+        title: 'Age verification',
+        acrValue: 'urn:age-verification',
+        scopes: [
```
Comment (jlndk) [inline src/utils/auth-methods.ts:125]: At the moment this does not match the available scopes on master.

The current ones are: 15, 16, 18, 21 (https://github.com/criipto/extensions-service/blob/master/extensions/ageverification/index.ts#L337)

**Comment by nmoskaleva** [comment]
@mickhansen @jlndk Updated the docs, please take a look. 
Will extensions documentation eventually go to a dedicated section (separate from _Guides & Tools_)?

**Comment by mickhansen** [comment]
> @mickhansen @jlndk Updated the docs, please take a look. Will extensions documentation eventually go to a dedicated section (separate from _Guides & Tools_)?

@nmoskaleva I added a small comment regarding "Your own MitID".
Not sure yet how we'll structure documentation, in the short term we just need somewhere to send age verification  clients.

**Review by nmoskaleva — APPROVED** [review]

**Pull request merged**